EventFilter/FEDInterface/interface/FED1024.h

```diff
@@ -112,27 +112,36 @@ namespace evf{
 
     class TCDSBST{
     public:
-      struct bst{
-	uint32_t gpstimelow;
-	uint32_t gpstimehigh;
-	uint32_t low0;
-	uint32_t high0;
-	uint32_t low1;
-	uint32_t high1;
-	uint32_t low2;
-	uint32_t high2;
-	uint32_t low3;
-	uint32_t high3;
-	uint32_t low4;
-	uint32_t high4;
-	uint32_t low5;
-	uint32_t status;
+      union tcdsbst {
+        uint64_t words[8];
+        struct {
+          uint32_t gpstimelow;
```

Comment:
before signoff I'd like to understand the memory alignment correctness of the union {uint64 words [8], struct {many things}}